### PR TITLE
add cis

### PR DESCRIPTION
**Pull request by matttrach**
replace rpm with cis example, upgrade server mods in example, update examples

### CODE REVIEW

File: examples/cis/config.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+write-kubeconfig-mode: 600
+cni: calico
+profile: "cis-1.23"
```
Comment (VestigeJ) [inline examples/cis/config.yaml:3]: the ` profile: cis ` is the newer syntax for this to avoid hitting deprecations

File: examples/cis/main.tf [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  user                = local.username
+  ssh_key             = local.public_ssh_key
+  ssh_key_name        = local.key_name
+  subnet_name         = "default"
+  security_group_name = module.aws_access.security_group_name
+}
+
+module "config" {
+  source            = "rancher/rke2-config/local"
+  version           = "v0.0.5"
```
Comment (VestigeJ) [inline examples/cis/main.tf:44]: bump to "v0.0.8" ?

**Review by VestigeJ — CHANGES_REQUESTED** [review]
Just two potential fixes ✔️

File: examples/cis/main.tf (hunk 2 above)
Comment (matttrach) [reply examples/cis/main.tf:44]: the config module is only on v0.0.5: https://registry.terraform.io/modules/rancher/rke2-config/local/latest
https://github.com/rancher/terraform-local-rke2-config

File: examples/cis/config.yaml (hunk 1 above)
Comment (matttrach) [reply examples/cis/config.yaml]: fixed, thank you!

**Review by VestigeJ — APPROVED** [review]
LGTM

**Pull request merged**